server/src/test/java/org/elasticsearch/search/aggregations/bucket/composite/CompositeAggregatorTests.java

```diff
@@ -1425,7 +1442,7 @@ public void testWithHistogramAndKeyword() throws IOException {
                     )
                 ).aggregateAfter(createAfterKey("histo", 0.8d, "keyword", "b"))
             , (result) -> {
-                assertEquals(3, result.getBuckets().size());
+                //assertEquals(3, result.getBuckets().size());
```

Comment:
Is this commented out in purpose?

Reply:
Good catch, it should be restored

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/48399/commits/95820b1a0bc6ee576ffce19bbf440c90ecb8a7a2